frontend/projects/upgrade/src/app/app.module.ts

```diff
@@ -69,10 +40,6 @@ export const getEnvironmentConfig = (http: HttpClient, env: Environment) => {
   ],
   providers: [
     { provide: ENV, useValue: environment },
-    provideAppInitializer(() => {
-      const initializerFn = getEnvironmentConfig(inject(HttpClient), inject(ENV));
```

Comment:
Both `inject()` and `provideAppInitializer()` are now unused and can be removed from the imports